github_app.go

```diff
@@ -205,25 +205,39 @@ func getInstallationIDFromCache(ctx context.Context, app int64, owner, repo stri
 	return installation.InstallationID, nil
 }
 
+type githubClient struct {
+	*github.Client
+}
+
+func (g *githubClient) Close() {
```

Comment:
sorry, quick comment. Can we name this something else, since it doesn't close it?